README.md

```diff
@@ -917,6 +917,62 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   </details>
 
+* <a id='comments'></a>(<a href='#comments'>link</a>) **Comment blocks should use single-line comments (`//` and `///`)**, rather than multi-line comments (`/* ... */` and `/** ... */`). [![SwiftFormat: blockComments](https://img.shields.io/badge/SwiftFormat-blockComments-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#blockComments)
```

Comment:
Suggest making it clear when each style is used
```suggestion
* <a id='comments'></a>(<a href='#comments'>link</a>) **Comment blocks should use single-line comments (`//` for code comments and `///` for documentation comments)**, rather than multi-line comments (`/* ... */` and `/** ... */`). [![SwiftFormat: blockComments](https://img.shields.io/badge/SwiftFormat-blockComments-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#blockComments)
```

Reply:
good idea